cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html

```diff
@@ -0,0 +1,58 @@
+
+{# ==========================================================================
+
+   image_text_25_75.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create an image and text 25/75 molecule.
+   See [GHE]/flapjack/Modules-V1/wiki/25-75-Image-&-Text
+
+   settings:
+
+       image_path: URL for image to display.
+
+       image_alt:  Alternative text for image.
+
+       heading:    Heading of the text content.
+
+       body:       Body of the text content.
+
+       link_url:   Link URL.
+
+       link_text:  Link text.
+
+   has_rule: Whether there is a bottom gray rule.
+             Default is false.
+
+   ========================================================================== #}
+
+{% macro render(settings, has_rule=false) %}
+<div class="m-image-text-25-75
+            {{ 'm-image-text-25-75__bottom-rule' if has_rule else '' }}">
+    <div class="m-image-text-25-75_image-container">
+
+    {# Use a div if this is a decorative image,
+       in which case the alt attribute is empty. #}
+    {% if settings.image_alt == '' %}
+        <div class="m-image-text-25-75_image u-centered-on-mobile"
+             style="background-image:url({{ settings.image_path }})"></div>
+    {% else %}
+        <img class="m-image-text-25-75_image u-centered-on-mobile"
+             src="{{ settings.image_path }}"
+             alt="{{ settings.image_alt }}">
+    {% endif %}
+    </div>
+    <div class="m-image-text-25-75_content">
+        <h3 class="m-image-text-25-75_heading">{{ settings.heading }}</h3>
+
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/f390b9e69fff8c84414077be04934aacd8745803